src/software/pipeline/main_LdrToHdrSampling.cpp

```diff
@@ -209,6 +220,45 @@ int aliceVision_main(int argc, char** argv)
             ALICEVISION_LOG_ERROR("Error while extracting samples from group " << groupIdx);
         }
 
+        using namespace boost::accumulators;
+        using Accumulator = accumulator_set<float, stats<tag::min, tag::max, tag::median, tag::mean>>;
+        Accumulator acc_nbUsedBrackets;
+        {
+            Histogram<int> histogram(1, usedNbBrackets, usedNbBrackets-1);
+            for(const hdr::ImageSample& sample : out_samples)
+            {
+                acc_nbUsedBrackets(sample.descriptions.size());
+                histogram.Add(sample.descriptions.size());
+            }
+            ALICEVISION_LOG_INFO("Number of used brackets in selected samples: "
+                                 << " min: " << extract::min(acc_nbUsedBrackets) << " max: "
+                                 << extract::max(acc_nbUsedBrackets) << " mean: " << extract::mean(acc_nbUsedBrackets)
+                                 << " median: " << extract::median(acc_nbUsedBrackets));
+
+            ALICEVISION_LOG_INFO("Histogram of the number of brackets per sample: " << histogram.ToString("", 2));
+        }
+        if(debug)
+        {
+            image::Image<image::RGBfColor> selectedPixels(width, height, true);
+
+            for(const hdr::ImageSample& sample: out_samples)
+            {
+                const float score = float(sample.descriptions.size()) / float(usedNbBrackets);
+                const Color color = getColorFromJetColorMap(score);
+                selectedPixels(sample.y, sample.x) = image::RGBfColor(color.r, color.g, color.b);
+            }
+            oiio::ParamValueList metadata;
+            metadata.push_back(oiio::ParamValue("AliceVision:nbSelectedPixels", int(selectedPixels.size())));
```

Comment:
emplace_back

Reply:
As noted in the commit message, I use push_back instead of emplace_back to avoid a strange link error on MSVC 2015.